apps/dashboard/src/app/(app)/(dashboard)/support/page.tsx

```diff
@@ -112,7 +118,26 @@ const HELP_PRODUCTS = [
   },
 ] as const;
 
-export default function SupportPage() {
+export default async function SupportPage() {
+  const [authToken, accountAddress] = await Promise.all([
+    getAuthToken(),
+    getAuthTokenWalletAddress(),
+  ]);
```

Comment:
what happens if the user is not logged in?

Reply:
nebula floating chat prompts the user to "sign in"